tsconfig.json

```diff
@@ -1,17 +1,12 @@
 {
   "compilerOptions": {
-    "declaration": true,
-    "declarationDir": "dist",
     "esModuleInterop": true,
+    "inlineSources": true,
     "module": "commonjs",
     "moduleResolution": "node",
-    "outDir": "dist",
-    "rootDir": "./src",
     "sourceMap": true,
-    "inlineSources": true,
     "strict": true,
     "target": "es6"
   },
-  "include": ["src/**/*.ts"],
-  "exclude": ["**/*.test.ts"]
+  "exclude": ["**/node_modules"]
```

Comment:
Nit: We can omit this. `node_modules` is excluded by default.

Reply:
Ah, right you are.